recipe/meta.yaml

```diff
@@ -36,7 +36,8 @@ requirements:
     - libpng
   run:
     - glfw
-    - octave
+    # Octave is missing a run_exports section
+    - {{ pin_compatible('octave', max_pin='x.x') }}
```

Comment:
Pin to compatible version. A run_exports section is being added to octave though, and this will soon not be required.